src/notification.cpp

```diff
@@ -52,6 +56,142 @@ NotificationManagerProxy *notificationManager()
     return notificationManagerProxyInstance();
 }
 
+QString encodeDBusCall(const QString &service, const QString &path, const QString &iface, const QString &method, const QVariantList &arguments)
+{
+    QString s;
+
+    s.append(service).append(' ');
+    s.append(path).append(' ');
+    s.append(iface).append(' ');
+    s.append(method);
```

Comment:
Yeah, I left it as it originally was, but since you just volunteered to review the change, I'll update it :)